cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIHDFSSinkTest.java

```diff
@@ -1196,5 +1212,352 @@ private Context createContext(String backendImpl, String backendMaxConns, String
+        headers.put(CommonConstants.HEADER_FIWARE_SERVICE_PATH, originalServicePath);
+        headers.put(NGSIConstants.FLUME_HEADER_MAPPED_SERVICE, mappedService);
+        headers.put(NGSIConstants.FLUME_HEADER_MAPPED_SERVICE_PATH, mappedServicePath);
+        //NGSICKANSink.ColumnAggregator columnAggregator = ngsickanSink.new ColumnAggregator(false, false, false, false);
+        NGSIHDFSSink.CSVRowAggregator rowAggregator = ngsihdfsSink.new CSVRowAggregator();
+        NotifyContextRequest.ContextElement contextElement = createContextElementForNativeTypes();
+        NGSIEvent ngsiEvent = new NGSIEvent(headers, contextElement.toString().getBytes(), contextElement, null);
+        rowAggregator.initialize(ngsiEvent);
+        rowAggregator.aggregate(ngsiEvent);
+        System.out.println(getTestTraceHead("[NGSIHDFSSink.testNativeTypeRowCsv]")
+                + "-  OK");
+        String correctAggregation = "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someNumber,number,2,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeNumberxffffnumber/someId_someTypexffffsomeNumberxffffnumber.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,somneBoolean,Boolean,true,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomneBooleanxffffBoolean/someId_someTypexffffsomneBooleanxffffBoolean.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someDate,DateTime,2016-09-21T01:23:00.00Z,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeDatexffffDateTime/someId_someTypexffffsomeDatexffffDateTime.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someGeoJson,geo:json,\"{type: Point,coordinates: [-0.036177,39.986159]}\",hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeGeoJsonxffffgeo:json/someId_someTypexffffsomeGeoJsonxffffgeo:json.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someJson,json,{String: string},hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeJsonxffffjson/someId_someTypexffffsomeJsonxffffjson.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someString,string,foo,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeStringxffffstring/someId_someTypexffffsomeStringxffffstring.txt\n" +
+                "1461136795,2016-04-20T07:19:55.801Z,somePath,someId,someType,someString2,string,,hdfs:///user/null/someServicesomePath/someId_someTypexffffsomeString2xffffstring/someId_someTypexffffsomeString2xffffstring.txt";
+        if (rowAggregator.getAggregation().equals(correctAggregation)) {
+            assertTrue(true);
+        } else {
+            fail();
+            System.out.println(getTestTraceHead("[NGSIHDFSSink.testNativeTypeRowCsv]")
+                    + "-  FAIL");
+        }
+
+    }//testNativeTypeRowCsv
```

Comment:
```suggestion
    } // testNativeTypeRowCsv
```

Reply:
Fixed in [9e61357](https://github.com/telefonicaid/fiware-cygnus/pull/1799/commits/9e61357a49559bffebc01a71389c87be49ebd72c)